catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -600,6 +633,30 @@ public static DAG convertResult(Result result, ORB orb, POA poa, List<String> re
             }
         }
 
+        if (!classificationAdded) {
+            //Add defaults from NSILI
+            addStringAttribute(graph,
+                    metadataSecurityNode,
+                    NsiliConstants.CLASSIFICATION,
+                    NsiliClassification.NO_CLASSIFICATION.getSpecName(),
+                    orb);
+            addedAttributes.add(buildAttr(attribute, NsiliConstants.CLASSIFICATION));
+
+            addStringAttribute(graph,
+                    metadataSecurityNode,
+                    NsiliConstants.POLICY,
+                    NsiliConstants.NATO,
+                    orb);
+            addedAttributes.add(buildAttr(attribute, NsiliConstants.POLICY));
+
+            addStringAttribute(graph,
+                    metadataSecurityNode,
+                    NsiliConstants.RELEASABILITY,
+                    NsiliConstants.NATO,
+                    orb);
+            addedAttributes.add(buildAttr(attribute, NsiliConstants.RELEASABILITY));
+        }
```

Comment:
For releasability, the spec says "Default value should be NATO."
For policy, the spec doesn't specify a default, so I set it to NATO as a default. 
Classification, I defaulted to "NO CLASSIFICATION" to represent that we didn't have something set.